lib/heartcheck/executors/base.rb

```diff
@@ -0,0 +1,15 @@
+module Heartcheck
+  module Executors
+    class Base
+      def dispatch(checkers)
+        checkers.map do |checker|
+          started = Time.now
+          checker.check.tap do |checked|
+            checked[:time] = (Time.now - started) * 1000.0
+            Logger.info Oj.dump(checked)
```

Comment:
One line per check make sense for me, see the sample below:

```
Started GET "/monitoring" for 127.0.0.1 at 2015-08-25 13:40:49 -0300
[Heartcheck] Start [Heartcheck::Controllers::Essential] from 127.0.0.1 at X
[Heartcheck] {"version":{"status":"ok"},"time":0.094121}
[Heartcheck] {"cache":{"status":"ok"},"time":2.819643}
[Heartcheck] {"cas-sys":{"status":"ok"},"time":163.749613}
[Heartcheck] {"cas-web":{"status":"ok"},"time":2.722345}
[Heartcheck] End [Heartcheck::Controllers::Essential]
```

With this mode, the entire response will appear in the log file, making easy to detect bottlenecks and another problems.